lib/components/Label.js

```diff
@@ -12,7 +13,18 @@ const Label = ({
   helpIconProps = null,
   ...otherProps
 }) => {
-  const { onClick, icon, ...otherHelpIconProps } = helpIconProps || {};
+  const { onClick, icon, href, to, tooltipProps, className: helpIconClassName, ...otherHelpIconProps } =
+    helpIconProps || {};
+  let Parent, elementSpecificProps;
+  if (href) {
+    Parent = "a";
+    elementSpecificProps = { href };
+  } else if (to) {
+    Parent = Link;
+    elementSpecificProps = { to };
+  } else {
+    Parent = "span";
+  }
```

Comment:
@ajmaln Let's not have this implementation that we have for the button component. It feels like overkill. Please remove. 
In case if there is such a requirement in the future, then `onClick` can handle the same.

Reply:
@edwinbbu Can we remove the `href` and `to` props also?